contracts/xsgd_contract.scilla

```diff
@@ -370,11 +370,10 @@ transition lawEnforcementWipingBurn(address : ByStr20, initiator : ByStr20)
       | Some b =>
         balances[address] := zero;
         currentSupply <- totalSupply;
-        new_supply = builtin sub currentSupply bal;
+        new_supply = builtin sub currentSupply b;
```

Comment:
@AmritKumar this is fixed in https://github.com/Xfers/XSGD-scilla/pull/13/files so I'm removing this from this PR

Reply:
```suggestion
        new_supply = builtin sub currentSupply bal;
```